entries/QUnit.module.xml

```diff
@@ -87,5 +124,43 @@ QUnit.test( "makes a car", function( assert ) {
 
 ]]></code>
 	</example>
+	<example>
+		<desc>Hooks stack on nested modules</desc>
+<code><![CDATA[
+QUnit.module( "grouped tests argument hooks", function( hooks ) {
+	var beforeEach = hooks.beforeEach;
```

Comment:
> Above the argument is called test, now its called hooks, that doesn't make much sense to me.

The previous example was wrong. The correct name is hooks and it does not contains the `test` function.

> Also, why do you add these variables? Can you just call hooks.beforeEach below?

A clean way. It's biased, thou. I was thinking on ES6 destructuring: `( { beforeEach, afterEach } )`.

Fixed at: 4ad0d98